food-truck-frontend/src/components/ReviewsList.tsx

```diff
@@ -0,0 +1,105 @@
+import React, {Component} from 'react';
+import api from "../util/api";
+
+const reviewsJSON = [{
+    "id": "",
+    "userId": "",
+    "truck": {"id": "",
+        "userId": "",
+        "name": "",
+        "menu": "",
+        "textMenu": "",
+        "priceRating": "",
+        "description": "",
+        "schedule": "",
+        "foodCategory": ""},
+    "starRating": "",
+    "costRating": "",
+    "reviewText": "",
+    "day_time": ""
+}];
+
+type Props = {
+    username: string
+}
+
+type State = {
+    data : {
+        "id": string,
+        "userId": string,
+        "truck": {"id": string,
```

Comment:
Just a note, the quotes around fields in the JSON objects aren't needed.
Also, once things are merged you should be able to use my `TruckState` for your truck field.